src/main/java/seedu/meeting/model/meeting/UniqueMeetingList.java

```diff
@@ -107,4 +58,14 @@ public boolean equals(Object other) {
                 || (other instanceof UniqueMeetingList // instanceof handles nulls
                 && internalList.equals(((UniqueMeetingList) other).internalList));
     }
+
+    /**
+     * Comparator for {@code Meeting} to sort meetings by time.
+     */
+    static class MeetingComparator implements Comparator<Meeting> {
```

Comment:
I remember that meeting already have a built in comparator

Reply:
Resolved in latest commit

Reply:
ok